installer - InstallerParamsTestCase.test_installer_options_and_flags - needs updating

the following option has been added:
```
E       AssertionError: Items in the second set but not the first:
E       u'--disable-resolve-mismatches'
```

--- a/tests/foreman/installer/test_installer.py
+++ b/tests/foreman/installer/test_installer.py
@@ -205,7 +205,8 @@ INSTALLER_OPTIONS = set([
     u'--foreman-proxy-dns-managed', u'--katello-max-tasks-per-pulp-worker',
     u'--foreman-group', u'--foreman-proxy-ssldir', u'--upgrade',
     u'--certs-regenerate', u'--capsule-puppet',
-    u'--foreman-plugin-discovery-install-images'
+    u'--foreman-plugin-discovery-install-images',
+    u'--disable-resolve-mismatches'
 ])
 
 

